docs/custom_service.md

```diff
@@ -89,6 +90,17 @@ class ModelHandler(object):
         :return: list of predict results
         """
         return ["OK"] * self._batch_size
+        
+    def handler(self, data, context):
```

Comment:
Call this handle()

Reply:
Yeah makes sense, I thought of differentiating it a bit, and I agree it can be a dummy